frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -73,14 +74,14 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
       <ModalBody>
         Increase the capacity of <strong>{ocsConfig.metadata.name}</strong>.
         <div className="add-capacity-modal--padding">
-          <div className="form-group">
+          <div className="add-capacity-modal__input form-group">
             <label className="control-label" htmlFor="request-size-input">
-              Requested Capacity
+              Capacity
               <FieldLevelHelp>{labelTooltip}</FieldLevelHelp>
               <span className="add-capacity-modal__span">
                 <span>
-                  Provisioned Capacity:
-                  {presentCount ? ` ${presentCount / 3}Ti` : ' Unavailable'}
+                  Provisioned:
+                  {presentCount ? ` ${presentCount * 2}TiB` : ' Unavailable'}
```

Comment:
shouldn't `Unavailable` be `text-secondary` ?

Reply:
ack